bin/labelcat.js

```diff
@@ -11,20 +11,16 @@ require(`yargs`)
     {},
     async opts => {
       util.makeCSV(
-        await util.retrieveIssues(opts.repoDataFilePath),
+        await util.retrieveIssues(opts.repoDataFilePath, 'type: bug'),
```

Comment:
I've seen a lot of issues use a `type: bug` label, but I've also seen a lot of issues use GitHub's default `bug` label. I think somehow you need to treat `bug` and `type: bug` as the same thing. Perhaps augment `retrieveIssues` to take an optional `alternatives` argument, which would be an array of strings, e.g. `await util.retrieveIssues(opts.repoDataFilePath, 'type: bug', ['bug'])`, where everything that has `bug` is treated as if it has `type: bug`.

Otherwise LGTM